README.md

```diff
@@ -2,3 +2,24 @@
 
 [![Build Status](https://travis-ci.org/dremio-hub/dremio-hbase-connector.svg?branch=master)](https://travis-ci.org/dremio-hub/dremio-hbase-connector)
 
+## Usage Instructions
+
+* From the project root run `mvn clean install`. The hbase connector jar should now be present
```

Comment:
Consider explicitly naming the .jar file.

Reply:
Done